Lib/test/test_inspect/test_inspect.py

```diff
@@ -176,6 +179,7 @@ def test_excluding_predicates(self):
         self.istest(inspect.ismethod, 'git.argue')
         self.istest(inspect.ismethod, 'mod.custom_method')
         self.istest(inspect.ismodule, 'mod')
+        self.istest(inspect.ispackage, 'pkg')
```

Comment:
Can you please also test several real python stdlib packages here? Like `asyncio` and `json`

Reply:
Please, also test that there are modules that are not packages. And that there things that are niether modules nor packages (like `1`) :)